src/Plugin/Condition/GroupMembership.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Drupal\og\Plugin\Condition;
+
+use Drupal\Core\Condition\ConditionPluginBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+
+/**
+ * Provides an 'OG Membership' block visibility condition.
+ *
+ * @Condition(
+ *   id = "og_group_membership",
+ *   label = @Translation("Group Membership"),
+ *   context = {
```

Comment:
Providing context definitions via the "context" key is deprecated in Drupal 8.7.x and will be removed before Drupal 9.0.0. Use the "context_definitions" key instead.

```suggestion
 *   context_definitions = {
```